buildroot-external/overlay/base/etc/init.d/S04CheckFactoryReset

```diff
@@ -51,7 +51,9 @@ check_factory_reset() {
     # mount /usr/local again
     mount /usr/local
   else
+    # clean whole /usr/local on oci platform
     rm -rf /usr/local
```

Comment:
actually this fix was bad - it should delete the content of the volume, not the volume itself - will fix

Reply:
sure, but `rm -rf /usr/local/*` will omit dotfiles, so if you fix it yourself, keep this in mind ;)